AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -32,6 +32,17 @@ public protocol APIClientProtocol: AnyObject {
     
 }
 
+/// :nodoc:
```

Comment:
It is ok to document it, btw :)
`AdyenNetworking` is a standalone lib now

Reply:
i think we should to through all of the `:nodoc` stuff, and see which variables can be `private` now. maybe in another PR